docs/source/architecture/permissioning_requirements.rst

```diff
@@ -0,0 +1,421 @@
+**************************
+Permissioning Requirements
+**************************
+
+The following is a detailed list of permissioning requirements, capabilities,
+and user stories that aid in the explanation and discussion of the
+permissioning design for Hyperledger Sawtooth.
+
+The following permission groups are presented in this document:
+
+  - Transactor key permissioning, which controls acceptance of transactions and
+    batches based on signing keys.
+  - Validator key permissioning, which controls which nodes are allowed to
+    establish connections to the validator network.
+
+The permissioning types are further broken down into capability requirements.
+Each requirement is made up of a short description, the network scenario the
+requirement supports, and associated user stories. User stories are short
+statements about a requirement or capability that an actor desires. These short
+descriptions illustrate the need for specific permission requirements.
+
+The design requirements for permissioning include both local validator
+configuration and network-wide on-chain permissioning. Local configuration is
+needed so that a validator can limit who is allowed to submit batches and
+transactions directly to that validator. This is useful, for example, if a
+company is running its own validator and wishes to only allow members of that
+company to submit transactions. On-chain configuration is required so that the
+whole of the network can enforce consistent permissioning rules. For example,
+in validator key permissioning, all the validators on the network will reject
+the same node from peering, ensuring that only known nodes can join the network.
+
+Definitions
+===========
+
+General Definitions
+-------------------
+
+The following define general components and concepts related to Sawtooth:
+
+Batch
+  A batch is a protobuf object containing a list of transactions, a header, and
+  a signature. The signature is generated by the batch signer by signing the
+  batch header with the batch signer's private key. For more information on the
+  structure of a batch, see the Architecture documentation.
```

Comment:
Instead of "see the Architecture documentation", sphinx should be able to find another section, so `Batch Data Structure` should work.

Reply:
Using back ticks is what I meant. But github markdown uses them, too.